department-of-veterans-affairs/va.gov-cms
# VACMS-2861: entity meta display block updates

## Description

See #2861 

## Testing done

Added behat test, manually tested.

## Screenshots

<img width="414" alt="Screen Shot 2020-10-21 at 3 21 45 PM" src="https://user-images.githubusercontent.com/101649/96791494-d47e7e00-13b5-11eb-8e1d-3b8f40a40654.png">

## QA steps

As a user with the content admin role:
1. Create a node without publishing it
1. [x] verify that the va.gov URL is not shown in the block
1. Publish the node
1. [x] verify that the va.gov URL is shown in the block, non-clickable with (Pending) appended
1. Archive the node
1. [x] verify that the va.gov URL is not shown in the block
1. Visit a node that has been previously published on production
1. [x] verify that the va.gov URL shows in the block and is clickable
1. Visit a revision page for a node
1. [x] verify that the block is shown

## Definition of Done
- [ ] Product release notes are written (or in progress), if required.
